docker_scripts/squash.py

```diff
@@ -518,8 +520,8 @@ def run(self):
             # without loading into Docker
             self._tar_image(self.output_path, new_image_dir)
             self.log.info("Image available at '%s'" % self.output_path)
-        else:
-            # Load image into Docker
+        if not self.output_path or self.load_output_back:
```

Comment:
Heh, wait a second, you can't use elif, since that would mean only the tar *or* the engine image would get stored :)

The logic needs to be something like this:
```
if self.output_path:
    # output tar
if self.load_output_back:
    # output engine image
if neither tar nor image were saved:
    # log warning
```

So logging the warning requires either a more complex conditions (`not self.output_path and not load_output_back`) or it requires a variable like `was_saved`, which would be initialized to `False` and set to `True` in the two first ifs... :) Since you probably wouldn't like the condition, I'm going to go with the `was_saved` variable, does that sound good?